### ISSUE TITLE
front_enqueue_js not working

### ISSUE THREAD

**Issue by Maden-maxi** [Theme-Fusion/Fusion-Builder-Sample-Add-On#9]
When I try include js file to my shortcode in the map function 
```
fusion_builder_map(
		array(
			'name' => esc_attr__('Related posts', 'fusion-builder'),
			'shortcode' => 'fusion_related_posts',
			'icon' => 'fa fa-archive',
			'preview' => SAMPLE_ADDON_PLUGIN_DIR . 'js/preview/related-posts-preview.php',
			'preview_id' => 'fusion-builder-block-module-related-posts-addon-preview-template',
			'front_enqueue_js' => SAMPLE_ADDON_PLUGIN_DIR .   'js/related-posts.js',
			'params' => array(
				array(
					'type'        => 'range',
					'heading'     => esc_attr__( 'Columns', 'fusion-builder' ),
					'description' => esc_attr__( 'Select the number of columns to display.', 'fusion-builder' ),
					'param_name'  => 'columns',
					'value'       => '3',
					'min'         => '1',
					'max'         => '6',
					'step'        => '1',
				),
				array(
					'type'          => 'colorpicker',
					'heading'    	=> __( 'Load More Button Background Color', 'fusion-builder' ),
					'param_name'    => 'color_btn',
					'value'         => '#47a3da',
					'description'   => __( 'Set the button title color.', 'fusion-builder' ),
				),
				array(
					'type'          => 'colorpicker',
					'heading'    	=> __( 'Load More Button Color Text', 'fusion-builder' ),
					'param_name'    => 'color_btn_text',
					'value'         => '#ffffff',
					'description'   => __( 'Set the button title color.', 'fusion-builder' ),
				),
			)
		)
	);
```
File not include with shortcode

**Comment by maximus80** [comment]
@Maden-maxi we are working on this currently and it will be fixed soon, sorry for the inconvenience.

@Theme-Fusion/developers  the elements in fusion_load_element_frontend_assets() are an empty array, so the enqueuing isn't working. We'll have to refactor this.

**Comment by lukeabeck** [comment]
Is this important for 5.1?  Not sure if this was moved over to main repo or not.

**Comment by MichaelWeb** [comment]
Should be fixed in 5.1.

**Comment by yawalkar** [comment]
@Maden-maxi To get this working for you until we release 5.1, please do the following change in Fusion Builder plugin - 

in `Fusion-Builder/inc/shortcodes.php` in the function `fusion_load_element_frontend_assets` replace the variable `$fusion_builder_elements` with `$all_fusion_builder_elements`

Like this - http://prntscr.com/ecwnfl

Closing this for now.
